### PR TITLE
Add DeleteEventsCompleted in signal handler for logging consistency #26606

### PR DESCRIPTION
**Pull request by helena**
#26606 minor and some cleanup.

### CODE REVIEW

File: akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventSourcedBehaviorImpl.scala [hunk 1]
```diff
@@ -90,19 +90,21 @@ private[akka] final case class EventSourcedBehaviorImpl[Command, Event, State](
...
+        ctx.log.debug("Persistent snapshots given criteria [{}] deleted successfully.", criteria)
       case DeleteSnapshotsFailed(DeletionTarget.Individual(meta), failure) =>
         ctx.log.warning("Failed to delete snapshot with meta [{}] due to [{}].", meta, failure)
       case DeleteSnapshotsFailed(DeletionTarget.Criteria(criteria), failure) =>
         ctx.log.warning("Failed to delete snapshots given criteria [{}] due to [{}].", criteria, failure)
+      case DeleteEventsCompleted(toSequenceNr) =>
+        ctx.log.debug("Events successfully deleted to sequence number [{}].", toSequenceNr)
       case DeleteEventsFailed(toSequenceNr, failure) =>
-        ctx.log.warning("Failed to delete messages toSequenceNr [{}] due to [{}].", toSequenceNr, failure)
+        ctx.log.warning("Failed to delete events to sequence number [{}] due to [{}].", toSequenceNr, failure)
```
Comment (helena) [inline akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventSourcedBehaviorImpl.scala]: yes/no?

Comment (patriknw) [reply akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventSourcedBehaviorImpl.scala]: yes

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by helena** [comment]
Merging with one review LGTM from @patriknw only because this is very small with minimal change and Patrik's approval.

**Pull request merged**